.pre-commit-config.yaml

```diff
@@ -19,6 +19,7 @@ repos:
     rev: v0.0.244
     hooks:
     -   id: ruff
+        args: [--fix, --exit-non-zero-on-fix]
```

Comment:
can we not add this please? there's a few false positives which risk confusing contributors

Reply:
Okay. Would remove it.